test/test_binary_ufuncs.py

```diff
@@ -573,56 +573,45 @@ def _do_pow_for_exponents(self, m1, exponents, pow_fn, atol):
                 self.assertEqual(res1, res2)
                 self.assertEqual(res1.dtype, expected_dtype)
 
-    def test_pow(self, device):
+    # not testing complex types due to https://github.com/pytorch/pytorch/pull/36793
```

Comment:
Would you elaborate on this? I'm not sure how the linked pull request means this test shouldn't test complex inputs.

Reply:
I had simply moved this comment but I'll check it out, as things might've changed since & it might now be possible to test on complex types.

Reply:
Oops, I had misread the comment I was supposed to move. This test should be enabled on complex tensors now.

Reply:
Done